.editorconfig

```diff
@@ -7,6 +7,6 @@ indent_size = 2
 tab_width = 2
 end_of_line = lf
 insert_final_newline = true
-trim_trailing_whitespace = true
+trim_trailing_whitespace = false
```

Comment:
This was causing whitespace to be trimmed in the expect strings in `expected` diffs in the test spec.

Those spaces are required as part of the signcolumn of the patches.

I thought I (once again) had a plugin that was interfering and auto formatting, always blaming my config :sweat_smile:

Reply:
Ahhhhhhhhhhhh of course. Alright. But, scouts honour, no trailing whitespace ;)